content/press/press-content.md

```diff
@@ -1,11 +1,34 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-**Press:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
 
-## Images from events
+## Resources for press/media personnel
+**For inquiries and information:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
+
+**Images from events**
 ![EEW public workshop participants](./eew-public-workshop.png)
 
-## Logos
+**Logos**
```

Comment:
It would probably be better to switch these to `###` (h3) than to just make them bold

Reply:
That's my preference but it's not a required change